inc/class-stream-wrapper.php

```diff
@@ -10,8 +10,9 @@
 use Exception;
 use GuzzleHttp\Psr7;
```

Comment:
Looks like this needs an ignore too?
```suggestion
use GuzzleHttp\Psr7; //phpcs:ignore -- Used in Psalm types
```